tests/integration/telemetry/stats/prometheus/http/wasm/stats_wasm_filter_test.go

```diff
@@ -48,6 +48,6 @@ func setupConfig(cfg *istio.Config) {
 		return
 	}
 	// enable telemetry v2 with Wasm
-	cfg.Values["telemetry.v2.metadataExchange.wasmEnabled"] = "true"
-	cfg.Values["telemetry.v2.prometheus.wasmEnabled"] = "true"
+	cfg.Values["telemetry.v2.metadataExchange.wasmEnabled"] = "false"
```

Comment:
@rshriram This test has been running fine for a long time.
It tests wasm in the telemetry specific tests.

We just need to disable compiled wasm in "default" profile.